Webapi/Rest/Request/Deserializer/XWwwFormUrlencoded.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Reach\Payment\Webapi\Rest\Request\Deserializer;
+
+
+use Magento\Framework\App\State;
+use Magento\Framework\Phrase;
+use Reach\Payment\Model\PostbackNotification\Decoder;
+
+class XWwwFormUrlencoded implements \Magento\Framework\Webapi\Rest\Request\DeserializerInterface

```

Comment:
A lot of this content follows the default `Deserializer` classes native to Magento. Reference: https://github.com/pepe1518/magento2/blob/master/vendor/magento/framework/Webapi/Rest/Request/Deserializer/Json.php